zerver/lib/actions.py

```diff
@@ -272,9 +272,9 @@ def send_signup_message(sender: UserProfile, admin_realm_signup_notifications_st
         "stream",
         admin_realm_signup_notifications_stream,
         user_profile.realm.display_subdomain,
-        "%s <`%s`> just signed up for Zulip!%s(total: **%i**)" % (
+        "@_**%s|%d** just signed up for Zulip!%s(total: **%i**)" % (
             user_profile.full_name,
-            user_profile.email,
+            user_profile.id,
```

Comment:
This hunk is wrong, because it's sent to the admin realm (so it's a cross-realm message).

Reply:
(Changed this to update the right message instead)